lambda_code/src/models/fhir_immunization_post_validators.py

```diff
@@ -34,17 +32,14 @@ def validate_and_set_vaccination_procedure_code(cls, values: dict) -> dict:
         field_location = generate_field_location_for_extension(url, system, field_type)
 
         try:
-            vaccination_procedure_code = get_generic_extension_value_from_model(
-                values, url, system, field_type
-            )
+            vaccination_procedure_code = get_generic_extension_value_from_model(values, url, system, field_type)
             if vaccination_procedure_code is None:
                 raise KeyError
-            cls.vaccine_type = PostValidation.vaccination_procedure_code(
-                vaccination_procedure_code, field_location
-            )
+            cls.vaccine_type = PostValidation.vaccination_procedure_code(vaccination_procedure_code, field_location)
 
         except (KeyError, IndexError, TypeError) as error:
             raise MandatoryError(f"{field_location} is a mandatory field") from error
+        cls.vaccine_type = PostValidation.vaccination_procedure_code(vaccination_procedure_code, field_location)
```

Comment:
Why has this line been added? Is a duplicate of line 38

Reply:
I think it's an artefact from a merge from master. removed.